dev/com.ibm.ws.javaee.ddmodel/test/com/ibm/ws/javaee/ddmodel/app/AppTest.java

```diff
@@ -17,58 +17,55 @@
 import com.ibm.ws.javaee.ddmodel.DDParser;
 import com.ibm.ws.javaee.version.JavaEEVersion;
 
-
 /**
  * Application deployment descriptor parse tests.
  */
 public class AppTest extends AppTestBase {
-    public static final Version[] TEST_VERSIONS =
-        new Version[] { 
-            JavaEEVersion.VERSION_6_0,
-            JavaEEVersion.VERSION_7_0,
-            JavaEEVersion.VERSION_8_0
-        };
+    public static final Version[] TEST_VERSIONS = new Version[] {
+                                                                  JavaEEVersion.VERSION_6_0,
+                                                                  JavaEEVersion.VERSION_7_0,
+                                                                  JavaEEVersion.VERSION_8_0
+    };
 
     public static final int CMP_LT = -1;
     public static final int CMP_GT = +1;
-    public static final int CMP_EQ =  0;
+    public static final int CMP_EQ = 0;
 
     public static int asCmp(int cmpResult) {
-        return ( (cmpResult < 0) ? CMP_LT : ((cmpResult > 0) ? CMP_GT : CMP_EQ) );
+        return ((cmpResult < 0) ? CMP_LT : ((cmpResult > 0) ? CMP_GT : CMP_EQ));
     }
 
-    public static int[][] COMPARE_VERSION_RESULTS =
-        new int[][] {
-            { CMP_EQ, CMP_LT, CMP_LT },
-            { CMP_GT, CMP_EQ, CMP_LT },
-            { CMP_GT, CMP_GT, CMP_EQ }
-        };
+    public static int[][] COMPARE_VERSION_RESULTS = new int[][] {
+                                                                  { CMP_EQ, CMP_LT, CMP_LT },
+                                                                  { CMP_GT, CMP_EQ, CMP_LT },
+                                                                  { CMP_GT, CMP_GT, CMP_EQ }
```

Comment:
The matrix of expected results would likewise need updating.

Reply:
done